sql/core/src/main/scala/org/apache/spark/sql/execution/SparkSqlParser.scala

```diff
@@ -505,7 +505,12 @@ class SparkSqlAstBuilder extends AstBuilder {
           } else {
             None
           }
-          (Seq.empty, Option(name), props.toSeq, recordHandler)
+          val finalProps = if (!props.contains("field.delim")) {
+            props.toSeq ++ Seq("field.delim" -> "\t")
```

Comment:
what's the behavior now? which delimiter do we use by default before this PR?

Reply:
> what's the behavior now? which delimiter do we use by default before this PR?

No default we use '\u0001'